### PR TITLE
[zh-cn]: Update translation of String.blink()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#13975

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/string/blink/index.md [hunk 1]
```diff
@@ -5,32 +5,28 @@ slug: Web/JavaScript/Reference/Global_Objects/String/blink
...
-
-`blink()`方法将字符串嵌入 `<blink>` 标签中：`"<blink>str</blink>"`。
+一个以 `<blink>` 开始标签开头的字符串，接着是文本 `str`，然后是 `</blink>` 结束标签。
 
 ## 示例
 
-### 使用 `blink()` 函数
+### 使用 blink()
 
 下面的示例使用了字符串方法来修改字符串格式：
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/string/blink/index.md:26]:
```suggestion
以下示例使用已弃用的字符串方法来更改字符串的格式：
```

File: files/zh-cn/web/javascript/reference/global_objects/string/blink/index.md [hunk 2]
```diff
@@ -5,32 +5,28 @@ slug: Web/JavaScript/Reference/Global_Objects/String/blink
 
 {{JSRef}} {{deprecated_header}}
 
-**`blink()`** 方法创建一个字符串，其在 `<blink>str</blink>` 中嵌入字符串，这使得字符串在旧版浏览器中闪烁。
+{{jsxref("String")}} 的 **`blink()`** 方法创建一个字符串，其在 `<blink>str</blink>` 中嵌入字符串，这使得字符串在旧版浏览器中闪烁。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/blink/index.md:8]:
```suggestion
{{jsxref("String")}} 值的 **`blink()`** 方法创建一个字符串，其在 `<blink>str</blink>` 中嵌入字符串，这使得字符串在旧版浏览器中闪烁。
```

**Review by yin1999 — COMMENTED** [review]
one nit. Could we add "值" here. As this is an instance method.

**Review by yin1999 — APPROVED** [review]
👍

**Pull request merged**